Library/Formula/fpp.rb

```diff
@@ -0,0 +1,22 @@
+class Fpp < Formula
+  homepage "https://facebook.github.io/PathPicker/"
+  # To be changed -- just a sample tar for now
+  url "https://github.com/pcottle/example_for_pip/raw/master/dist/pcottleexample.0.0.4.tar.gz"
+  sha256 "25f3edddc28b7a353c407d48a74a92f9a364160de790122357a3785d9d9e8aeb"
+  head "https://github.com/facebook/pathpicker.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    puts buildpath
```

Comment:
You can remove this and the two `# ` comments below. It's all fairly regular Homebrew stuff that doesn't need special explanation :)